calebwagner/sink-repair
# created delete function for service requests

Steps to check code:
1. git fetch --all
2. git checkout -cjw-delete-btn
3. code .
4. serve
5. check in Requests.js module for an event listener
6. check for the HTML representation of the delete button within the Requests function
7. create a request in the browser 
8. delete the request you made in the browser
